src/kaleidoscope/device/avr/pins_and_ports.h

```diff
@@ -123,7 +123,65 @@
 #define PIN_F6 PINDEF(F, 6)
 #define PIN_F7 PINDEF(F, 7)
 #endif
-
+#else // ifndef KALEIDOSCOPE_VIRTUAL_BUILD
```

Comment:
Would it be easy enough to just redefine  PINDEF if KALEIDOSCOPE_VIRTUAL_BUILD is defined?  That'd cut down a lot on the repeated definitions here.

Reply:
Yep. Just force-pushed an update.